utilities.lib.nut

```diff
@@ -157,28 +183,20 @@ utilities._isLeapYear <- function(y) {
 // **********         Public           **********
 utilities.getNewUUID <- function(cb = null) {
     if (cb == null) {
-        server.error("getNewUUID() requires a callback function with err, data parameters");
-        return;
+        throw "getNewUUID() requires a callback function with err, data parameters";
```

Comment:
Why is cb an optional parameter?

Reply:
So that I can test for its inclusion without a Squirrel runtime error being generated.